pages/admin/permit-holders.tsx

```diff
@@ -18,43 +19,31 @@ import {
 } from '@chakra-ui/react'; // Chakra UI
 import { ChevronDownIcon, SearchIcon, WarningIcon, WarningTwoIcon } from '@chakra-ui/icons'; // Chakra UI Icons
 import Layout from '@components/internal/Layout'; // Layout component
-import { Role } from '@lib/types'; // Role enum
+import { Applicant, Permit, PermitStatus, Role, UserStatus } from '@lib/types'; // Role enum
 import { authorize } from '@tools/authorization'; // Page authorization
 import Table from '@components/internal/Table'; // Table component
 import Pagination from '@components/internal/Pagination'; // Pagination component
-import { useState, SetStateAction, Dispatch } from 'react'; // React
+import { useState } from 'react'; // React
+import {
+  FILTER_PERMIT_HOLDERS_QUERY,
+  FilterPermitHoldersRequest,
+  FilterPermitHoldersResponse,
+} from '@tools/pages/permit-holders/filter-permit-holders';
 import DayPicker, { DateUtils, DayPickerProps, RangeModifier } from 'react-day-picker'; // Date picker
 import 'react-day-picker/lib/style.css'; // Date picker styling
 import Helmet from 'react-helmet'; // Date picker inline styling for select range functionality
+import { SortOptions, SortOrder } from '@tools/types'; // Sorting types
+import { Column } from 'react-table'; // Column type for table
+import useDebounce from '@tools/hooks/useDebounce';
+import { useEffect } from 'react';
 
-// Placeholder data
-
-const DATA = Array(10).fill({
-  name: {
-    name: 'Charmaine Wang',
-    id: 36565,
-  },
-  dateOfBirth: '2021/01/01',
-  homeAddress: {
-    address: '5300, No.3 Rd Lansdowne Centre',
-    city: 'Richmond',
-    postalCode: 'V2X 1E4',
-  },
-  email: 'charmainewang@rcd.org',
-  phone: '000-000-0000',
-  recentAPP: {
-    appNumber: '#XXXXXX',
-    expiryDate: new Date(),
-  },
-  userStatus: 'active',
-});
-
-const COLUMNS = [
+const COLUMNS: Column<any>[] = [
   {
     Header: 'Name',
     accessor: 'name',
     width: 180,
     minWidth: 180,
+    sortDescFirst: true,
```

Comment:
Isn't this supposed to be `false`?

Reply:
mb forgot to respond to this one. The reason i have it false is bc the default for the name column is asc so clicking the column won't change anything the first time. I can remove it but i think it's better than making the user click the column twice to change order the first time

Reply:
the sortDescFirst option is for the first click btw the name is a bit deceiving